admin/services/business-availability.service.js

```diff
@@ -113,6 +113,7 @@ businessAvailabilityService.determineAccessArrangementsEligibility = (checkWindo
  */
 businessAvailabilityService.getAvailabilityData = async (dfeNumber, checkWindowData) => {
   const currentDate = moment.utc()
+  const isWithinOpeningHours = currentDate.hour() >= 8 && currentDate.hour() < 16
```

Comment:
@danwilliams - how will this play with your mod change?

Reply:
not very well given that we are not taking into account the timezone of the school.  the offset is available as part of the session object